css/styles.css

```diff
@@ -0,0 +1,680 @@
+* {
+  list-style: none;
+  text-decoration: none;
+}
+
+:root {
+  --black-color: #272a31;
+  --red-color: #ec5242;
+  --gray-color: #d3d3d3;
+  --nav-color: #f7f4f4;
+}
+
+a:hover {
+  cursor: pointer;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.bar {
+  display: block;
+  width: 43px;
+  height: 8px;
+  margin: 3px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #000;
+}
+
+.nav-link .campain {
+  border: #94544d solid;
+}
+
+.nav-ul {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 15px;
+}
+
+.logo img {
+  width: 10em;
+}
+
+.topbar img {
+  margin: -7px 0 4px 1px;
+}
+
+.nav-ul li {
+  display: flex;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  align-items: center;
+  text-align: center;
+  color: #272a31;
+  margin-left: 32px;
+}
+
+.cards {
+  display: flex;
+  gap: 1em;
+  justify-content: space-evenly;
+}
+
+.navbar-desktop {
+  display: flex;
+  justify-content: space-between;
+}
+
+.realbavlist {
+  display: flex;
+  gap: 3em;
+  margin: 3% 0% 0% -5%;
+}
+
+.firstpart {
+  background-image: url(../img/07.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: auto;
+}
+
+.text-container > p {
+  padding-top: 3em;
+  width: 90%;
+  margin: auto;
+  font-family: 'Lato', sans-serif;
+  font-size: x-large;
+  font-weight: 600;
+  color: var(--red-color);
+  text-align: center;
+}
+
+.creative-nexus {
+  padding-top: 1em;
+  padding-bottom: 1em;
+  width: 90%;
+  margin: auto;
+  font-size: xx-large;
+  font-weight: 900;
+  color: rgba(236, 82, 66, 0.9);
+  font-family: 'Cocogoose', sans-serif;
+}
+
+.boxed-paragraph {
+  display: block;
+  border: solid #fff;
+  border-radius: 3px;
+  padding: 1em;
+  width: 90%;
+  margin: auto;
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  color: var(--black-color);
+}
+
+.header-time {
+  padding-top: 1em;
+  width: 90%;
+  margin: auto;
+  font-family: 'Cocogoose', sans-serif;
+  font-size: xx-large;
+  font-weight: 900;
+  color: var(--black-color);
+}
+
+.below-h1 {
+  width: 90%;
+  margin: auto;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  color: var(--black-color);
+  padding-bottom: 5em;
+}
+
+.tesxth {
+  padding-top: 4em;
+  width: 90%;
+  margin: auto;
+  font-family: 'Lato', sans-serif;
+  font-size: x-large;
+  font-weight: 400;
+  color: var(--red-color);
+}
+
+.mainprogram {
+  background-color: var(--black-color);
+  padding-bottom: 2.5em;
+  width: 100%;
+  background-image: url(../img/00.png);
+}
```

Comment:
- [ ] Kindly make sure to align your main program contents correctly by providing the padding on both sides in order to meet the provided design. :wink: 
- [ ] Kindly make sure the typography of the contents the provided design.

  - ### YOUR DESIGN
   ![image](https://user-images.githubusercontent.com/87197412/203583745-75e25787-222e-4b32-931c-5f83d9d1de31.png)


   - ### PROVIDED DESIGN
   ![image](https://user-images.githubusercontent.com/87197412/203583666-03871f70-0361-4126-8dd9-88d99429fa1a.png)

Reply:
- Hey, great job making the changes suggested by the previous reviewer, however, he pointed out that you should use the correct font family, which you've done but you're still missing to use the right color for the description on each card supposed to be white but you've used black. I should have marked this *OPTIONAL* but since he pointed it out without specifying, he assumed you should make all the necessary changes regarding his review. Please let's update the color to white to match the design as much as possible.

*original design*
![Screenshot from 2022-11-24 15-50-59](https://user-images.githubusercontent.com/59397159/203814300-ab23882b-0342-493d-975d-07b40ad96d59.png)

*your design*
![Screenshot from 2022-11-24 16-00-28](https://user-images.githubusercontent.com/59397159/203814699-f50a8662-8bfe-4ddd-8533-bd10dffe8382.png)

Reply:
- [ ] This issue is not yet resolved :x: